cmd/ponyup.pony

```diff
@@ -94,16 +94,18 @@ actor Ponyup
       end
 
     let pkg' = (consume pkg).update_version(version)
+    let pkg_str: String val = pkg'.string()
 
     if _lockfile.contains(pkg') then
-      _notify.log(Info, pkg'.string() + " is up to date")
+      _notify.log(Info, pkg_str + " is up to date")
       return
     end
 
     (let install_path, let dl_path) =
       try
-        let p = _root.join(pkg'.string())?
-        (p, FilePath(_auth, p.path + ".tar.gz"))
+        let p = _root.join(pkg_str)?
+        (p, FilePath(_auth,
+          p.path + ifdef windows then ".zip" else ".tar.gz" end))
```

Comment:
i think this would be easier to read if the string was constructed outside the `FilePath` call.

Reply:
Fixed.